File: Level_1/하샤드 수/Harshad_number.java
class Solution {
    public boolean solution(int x) {
        boolean answer = true;
        int sum = 0, tmp = x;
        
        while(tmp != 0){
            sum += tmp % 10;
            tmp /= 10;
        }
        if(x % sum != 0) answer = false; 
        
        return answer;
    }
}

File: Level_1/하샤드 수/Harshad_number.js
function solution(x) {
    let answer = true;
    let sum = 0, tmp = x;
    
    while(tmp > 0){
        sum += tmp % 10;
        tmp = Math.floor(tmp/10);
    }
    answer = (x % sum) === 0 ? true : false;
    
    return answer;
}
